app/policies/division_owned_scope.rb

```diff
@@ -0,0 +1,7 @@
+class DivisionOwnedScope < ApplicationPolicy::Scope
+
+  def resolve
+    scope.where(division_id: user.accessible_division_ids)
```

Comment:
I'm not sure that you need to reach out of the Policy system and back into the user model here. Why not just construct the DivisionPolicy in this class directly instead of in User?